docs/integrations/saas-cloud/miro.md

```diff
@@ -0,0 +1,109 @@
+
+## Installing the Miro App
+
+This section has instructions for installing the the Miro App for Sumo Logic and descriptions of each of the dashboards.
+
+Locate and install the app you need from the **App Catalog**. If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
+
+1. From the **App Catalog**, search for and select the app.
+1. Click **Add Integration**.
+1. Configure **Miro App** using the steps described in the [Miro Cloud-to-Cloud Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/miro-source/). If you already have set up your data, skip this step and click **Next**.
+1. Complete the following fields.
+   1. **Data Source**. Select either of these options for the data source:
+      * Choose **Source Category** and then choose a source category from the list.
+      * Select **Enter a Custom Data Filter** and enter a custom source category that starts with an underscore. For example, `_sourceCategory=MyCategory`.
+    2. **Folder Name**. You can retain the existing name, or enter a name of your choice for the app.�
+    3. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+1. Click **Next**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. From here, you can share it with your organization.
+
+Panels will start to fill automatically. It's important to note that each panel slowly fills with data that matches the time range query and has been received since the panel was created. Results will not be available right away, but with a bit of time, you'll see full graphs and maps.
+
+## Viewing Miro Dashboards​
+
+### Miro - Overview
+
+The **Miro - Overview** provides a comprehensive overview of events related to Miro. The dashboard features various widgets, including Total Events, Total Board Events, Distribution of App Events, Events By Team, Boards Created and Opened over Time, Events Over Time, Geo-Location of Events, Recent File Event Summary, and Recent Event Summary.
+
+The widgets allow users to track and analyze event-related information, including the total number of events, their distribution, team-wise breakdown, creation and opening of boards over time, and their geographical locations. The Recent File Event Summary and Recent Event Summary widgets provide a summary of the latest events for quick reference. A dashboard is a useful tool for monitoring and improving event management by providing quick access to relevant information.
+
+<img src='https://sumologic-app-data-v2.s3.amazonaws.com/dashboards/Miro/Miro-Overview.png' alt="Miro-Overview.png" />
```

Comment:
@JV0812  please add this Dashboard image to AWS S3 as we discussed in Asana. Otherwise it won't show here or in the UI.

Reply:
App is still under QA, once it ready I will add the image to S3 bucket.